### PR TITLE
Begin merge into master - README and git files

### PR DESCRIPTION
**Pull request by gwaygenomics**
I will do this in bite-size chunks, porting over entire files instead of commits. 

The first PR into master will add the README, associated README figures enabling accurate rendering (no README scripts yet), and small `.git` related files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.ipynb_checkpoints/
+*__pycache__
+.Rhistory
+Rplots.pdf
+data/raw/
+data/mnist/
+
```
Comment (jaclyn-taroni) [inline .gitignore]: Extra line here?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,105 @@
...
+![VAE t-SNE](figures/tsne_vae.png?raw=true)
+
+### Training
+
+The current model training is explained in
+[this notebook](tybalt_vae.ipynb)
+
+For a complete pipeline with reproducibility instructions, refer to
+[run_pipeline.sh](run_pipeline.sh). Note that scripts originally written in
+Jupyter notebooks ported to the scripts folder for pipeline purposes with:
```
Comment (jaclyn-taroni) [inline README.md]: Should this be

> Note that scripts originally written in Jupiter notebooks _can be_ ported to the scripts for pipeline purposes

I feel like there are some words missing

File: README.md [hunk 3]
```diff
@@ -1,2 +1,105 @@
...
+#### Parameter sweep
+
+In order to select the most optimal parameters for the model, we ran a
+parameter search over a small grid of parameters. See
+[parameter_sweep.md](parameter_sweep.md) for more details. Overall, we selected
+optimal `learning rate = 0.0005`, `batch size = 50`, `epochs = 100`. Training
+with optimal parameters was similar for training and a 10% test set across each
+epoch.
+
+![Training Performance](figures/onehidden_vae_training.png?raw=true)
```
Comment (jaclyn-taroni) [inline README.md]: I think this figure needs a little more context here, ideally with a short legend. For example - is this the training performance for the final model? What do the two lines represent?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,105 @@
...
+
+#### Model Evaluation
+
+After training with optimal hyper parameters, the unsupervised model can be
+interpreted. For instance, the distribution of activations across different
+nodes can be visualized. For example, the first 10 nodes (of 100) can be
+visualized by sample activation patterns.
```
Comment (jaclyn-taroni) [inline README.md]: What is the goal of having this figure here? To demonstrate interpretability?
Is this the distribution of activations across samples? One sample? Hard to tell from the info flanking the figure itself.

File: README.md (hunk 2 above)
Comment (gwaygenomics) [inline README.md]: ah, yes, (some) words (are) missing

File: README.md (hunk 3 above)
Comment (gwaygenomics) [inline README.md]: Yes, I should add more context

File: .gitignore (hunk 1 above)
Comment (gwaygenomics) [inline .gitignore]: i think its ok, otherwise, github will yell at me

**Comment by gwaygenomics** [comment]
thanks for the comments @jaclyn-taroni - I believe I have addressed them in the most recent commits. Ready for review again

**Pull request merged**